src/terms/hartree.jl

```diff
@@ -11,22 +11,26 @@ For the Coulomb potential with periodic boundary conditions, this is rather
 where G is the Green's function of the periodic Laplacian with zero
 mean (-Δ G = sum_{R} 4π δ_R, integral of G zero on a unit cell).
 """
-struct Hartree end
-(::Hartree)(basis) = TermHartree(basis)
+struct Hartree
+    scaling::Real  # to scale by an arbitrary factor (useful for exploration)
+end
+Hartree() = Hartree(1)
+(H::Hartree)(basis) = TermHartree(basis, H.scaling)
```

Comment:
More than one letter, please.

Reply:
But the planet!

Reply:
In fact I'm not sure what would be a good name except for H here.

Reply:
term? operator? hartree?